cfgov/cfgov/settings/base.py

```diff
@@ -262,7 +262,7 @@
 PDFREACTOR_LIB = os.environ.get('PDFREACTOR_LIB', '/opt/PDFreactor/wrappers/python/lib')
 
 # S3 Configuration
-if os.environ.get('S3_ENABLED', False):
+if os.environ.get('S3_ENABLED', 'False') == 'True':
```

Comment:
Sorry I meant `eval()` but now that I'm thinking about it that's probably a bad idea, security wise.